src/ai/prompts/openai_prompts.py

```diff
@@ -843,8 +847,20 @@ def run(self, input: str, collection_id: str = None, llm_callbacks: list = [], a
 
 If there isn't enough context to judge a piece of code, do not comment on it.
 
+Include the "language" key in the output to specify the language of the source code file being reviewed. e.g.
+- C -> "c"
+- C++ -> "cpp"
+- Python -> "python"
```

Comment:
This is a good example of what you're talking about, I guess.. with regards to using a map instead of plain English.

Reply:
Yep